dht/range_streamer.cc

```diff
@@ -292,6 +291,8 @@ future<> range_streamer::stream_async() {
                     for (auto it = range_vec.begin(); it < range_vec.end();) {
                         ranges_to_stream.push_back(*it);
                         ++it;
+                        auto percentage = _db.local().get_config().stream_plan_ranges_batch_percentage();
```

Comment:
Will this get the most updated value in case it was live updated? (just to make sure)

Reply:
Yes, it is from the _db and it is live update.